src/components/Icon/Expensicons.js

```diff
@@ -72,10 +72,14 @@ import Wallet from '../../../assets/images/wallet.svg';
 import Workspace from '../../../assets/images/workspace-default-avatar.svg';
 import ActiveRoomAvatar from '../../../assets/images/avatars/room.svg';
 import DeletedRoomAvatar from '../../../assets/images/avatars/deleted-room.svg';
+import AdminsRoomAvatar from '../../../assets/images/avatars/admins-room.svg';
+import AnnounceRoomAvatar from '../../../assets/images/avatars/announce-room.svg';
 
 export {
     ActiveRoomAvatar,
+    AdminsRoomAvatar,
```

Comment:
Is it just me or AdminsRoomAvatar sounds kinda weird?
How about `AdminRoomAvatar`? @thesahindia

Reply:
Yeah I was going to use that but it's for #admins room so I thought AdminsRoomAvatar will be correct.

Reply:
No problem, let's change it to AdminRoomAvatar everywhere for now